app/javascript/dashboard/routes/dashboard/conversation/contact/ContactInfo.vue

```diff
@@ -80,11 +109,24 @@
         @cancel="toggleEditModal"
       />
       <new-conversation
+        v-if="contact.id"
         :show="showConversationModal"
         :contact="contact"
         @cancel="toggleConversationModal"
       />
     </div>
+    <woot-confirm-delete-modal
+      v-if="showDeleteModal"
+      :show.sync="showDeleteModal"
+      :title="$t('DELETE_CONTACT.CONFIRM.TITLE')"
+      :message="confirmDeleteMessage"
+      :confirm-text="deleteConfirmText"
+      :reject-text="deleteRejectText"
+      :confirm-value="contact.name"
+      :confirm-place-holder-text="confirmPlaceHolderText"
+      @on-confirm="confirmDeletion"
+      @on-close="closeDelete"
```

Comment:
Since @ already signifies an event we can rename this to `@close`

Reply:
Same for on-confirm to @confirm

Reply:
`woot-confirm-delete-modal` is common component. if we change @on-confirm to @confirm there are 4 places that need to be changed. 

Can we take this as a separate PR?